espanol/canudos.md

```diff
@@ -31,3 +31,5 @@ el Consejero:
 [Una bruja entró en el salón](bruja/bruja.md)
 
 [Subió a la torre](torre/torre.md)
+
+[Ordenó buscar al Capitán Trueno](capitan/trueno.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.